core/src/test/java/org/apache/iceberg/rest/responses/TestRESTCatalogConfigResponse.java

```diff
@@ -156,14 +156,6 @@ public void testDeserializeInvalidResponse() {
         () -> deserialize(jsonOverridesHasWrongType)
     );
 
-    String jsonMisspelledKeys =
-        "{\"defaultzzzzzzz\":{\"warehouse\":\"s3://bucket/warehouse\"},\"overrrrrrrrides\":{\"clients\":\"5\"}}";
-    AssertHelpers.assertThrows(
-        "A JSON response with the keys spelled incorrectly should fail to deserialize and validate",
-        JsonProcessingException.class,
-        () -> deserialize(jsonMisspelledKeys)
-    );
```

Comment:
Because the RESTObjectMapper does not fail on unknown keys, this test is no longer valid.

We might consider adding a warning log in the config response's `validate` method to log if `null` was received for both (not an explicit empty collection, but null).

In practice, users should be using the config response class and its associated builder, which cannot build an incorrect payload.